compiler/daml-extension/src/extension.ts

```diff
@@ -97,17 +97,44 @@ export async function activate(context: vscode.ExtensionContext) {
 }
 
 // Compare the extension version with the one stored in the global state.
-// This will be used to show release notes when the version has changed.
-async function checkVersion(context: ExtensionContext) {
+// If they are different, we assume the user has updated the extension and
+// we display the release notes for the new SDK release in a new tab.
+// This should only occur the first time the user uses the extension after
+// an update.
+async function showReleaseNotesIfNewVersion(context: ExtensionContext) {
     let packageFile = path.join(context.extensionPath, 'package.json');
     let packageData = await util.promisify(fs.readFile)(packageFile, "utf8");
     let extensionVersion = JSON.parse(packageData).version;
     let recordedVersion = context.globalState.get(versionContextKey);
-    if (!recordedVersion || recordedVersion != extensionVersion ) {
+    if (!recordedVersion || recordedVersion != extensionVersion) {
+        // We have a new version of the extension so show the release notes
+        // and update the current version so we don't show them again until
+        // the next update.
+        showReleaseNotes(extensionVersion);
         await context.globalState.update(versionContextKey, extensionVersion);
     }
 }
 
+// Show the release notes from the DAML Blog.
+// We display the HTML in a new editor tab using a "webview":
+// https://code.visualstudio.com/api/extension-guides/webview
+async function showReleaseNotes(version: String) {
+    let releaseNotesUrl = 'https://blog.daml.com/release-notes/';
```

Comment:
```suggestion
    const releaseNotesUrl = 'https://blog.daml.com/release-notes/';
```
Same for all `let` and `var` below.